pyall/analysis-2016_02/meta.yaml

```diff
@@ -1,23 +1,25 @@
+{% set version = "2016_02" %}
+
 package:
   name: analysis
-  version: '2016_01'
+  version: {{ version }}
 
 build:
-  number: 3
+  number: 0
 
 requirements:
   run:
-    - python
-    - jupyter
-    - ipywidgets
-    - pymongo
-    - datamuxer
-    - databroker
+    - bluesky  # [py3k]
     - channelarchiver
+    - databroker
+    - datamuxer
+    - ipywidgets
+    - jupyter
     - lmfit ==0.8.3
     - matplotlib
     - openpyxl
+    - pymongo
```

Comment:
> Would it be better to do that in the mds recipe?

Yeah.  Pinning in the library recipe where it actually matters is almost certainly a good policy